src/sentry/receivers/onboarding.py

```diff
@@ -412,7 +412,15 @@ def record_member_joined(organization_id: int, organization_member_id: int, **kw
 
 
 def record_release_received(project, event, **kwargs):
-    if not event.get_tag("sentry:release"):
+    event_type = event.get_event_type()
+
+    release = (
+        event.data.get("release")
+        if event_type == "transaction"
+        else event.get_tag("sentry:release")
```

Comment:
it is not clear to me why we are checking for the tag `sentry:release` if the `release` property is always present in data. @jan-auer do you happen to know why?